apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/EvaluationResults/EvaluationResultsTable.tsx

```diff
@@ -44,34 +45,33 @@ export const ResultCellContent = ({
   value,
 }: {
   evaluation: EvaluationDto
-  value: unknown
+  value: EvaluationResultDto['result']
 }) => {
+  if (value === undefined) {
+    return <Badge variant='secondary'>Pending</Badge>
+  }
+
   if (evaluation.resultType === EvaluationResultableType.Boolean) {
+    value = typeof value === 'string' ? value === 'true' : Boolean(value)
```

Comment:
hmm this defense shoult not be needed since the eval result column in db is a boolean. Why do we need it?/

Reply:
Where I use this, the result value comes already parsed as a number, boolean or string. However, looking at the original code of this component, it seemed it could come as a string: `{value === 'true' ? 'success' : 'destructive'}`, so I left this logic as is to not break things.